okonomiyaki/file_formats/_egg_info.py

```diff
@@ -220,12 +224,12 @@ def from_string(cls, s, strictness=2):
 
         Parameters
         ----------
-        s: str
-            Egg name, e.g. 'Qt-4.8.5-2'.
+        s: text type
```

Comment:
the convention is to call it text type I think (e.g. that's how `six` calls this). It is the type `unicode` on python 2, and the type `str` on python 3.